lib/OperatingSystems/DefaultOs.php

```diff
@@ -179,6 +179,9 @@ public function getNetworkInterfaces(): array {
 		return $result;
 	}
 
+	/**
+	 * @return array
+	 */
```

Comment:
```suggestion
```

Please remove the docblock. IOperatingSystem already contains the correct type hint.